docs/index.html

```diff
@@ -473,7 +473,7 @@ <h3>What is Apple Advanced Typography?<a class="headerlink" href="#what-is-apple
 reference) will bring up a dialog showing you what glyph is referred to, and
 allowing you to bring up an editing window on that glyph. Finally the
 <a class="reference internal" href="ui/menus/elementmenu.html#elementmenu-accented"><span class="std std-ref">Element-&gt;Build-&gt;Accented Glyphs</span></a> command will
-figure out what glyphs should be refered to to build this composite glyph, and
+figure out what glyphs should be refered to build this composite glyph, and

```

Comment:
This also was correct. But if you have to change something, how about the misspelled word? Didn't notice that?